j2lint/cli.py

```diff
@@ -101,36 +99,15 @@ def run(args=None):
 
     if not options.log and not options.vv and not options.vvv:
         logging.disable(sys.maxsize)
-    else:
-
-        if options.vv:
-            # Enable logs on console
-            logging.basicConfig(
-                level=logging.INFO,
-                format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
-                handlers=[
-                    logging.StreamHandler(sys.stdout)
-                ]
-            )
-        time.sleep(1)
-
-        if options.vvv:
-            # Enable debug logs on console
-            logging.basicConfig(
-                level=logging.DEBUG,
-                format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
-                handlers=[
-                    logging.StreamHandler(sys.stdout)
-                ]
-            )
-        time.sleep(1)
 
+    else:
+        stream_level = logging.INFO
```

Comment:
rename to log_level rather than stream_level since it is applicable to all handlers